var/spack/repos/builtin/packages/libtiff/package.py

```diff
@@ -55,6 +56,9 @@ class Libtiff(CMakePackage, AutotoolsPackage):
     version("4.0.4", sha256="8cb1d90c96f61cdfc0bcf036acc251c9dbe6320334da941c7a83cfe1576ef890")
     version("3.9.7", sha256="f5d64dd4ce61c55f5e9f6dc3920fbe5a41e02c2e607da7117a35eb5c320cef6a")
 
+    # GUI
+    variant("opengl", default=False, description="build tiffgt viewer", when="@4.5:")
```

Comment:
```suggestion
    variant("opengl", default=False, description="use OpenGL (required for tiffgt viewer)", when="@4.5,4.7:")
```
This one is weird. Looks like tiffgt has always existed. Starting with libtiff 4.5, an option was added to explicitly control it. Then in 4.6, this option was removed. However, on main, that PR was reverted. So I'm guessing this will come back?

Reply:
Indeed, it looks like it was briefly controlled by the "tiff-tools-unsupported" CMake option:
- The "tiff-tools-unsupported" option also includes `tiff2pdf` and `tiff2ps` per the diff https://gitlab.com/libtiff/libtiff/-/commit/9ab54a858049bef020d578c71d82669531551c00
(it's a large diff to page through).
- The ChangeLog shows the option was introduced on 2023-08-16 https://gitlab.com/libtiff/libtiff/-/blob/4e63559f2b7fa3ab5c8fa8ea0dbcc21e62286fe0/ChangeLog#L100 and here's the referenced issue with more detail https://gitlab.com/libtiff/libtiff/-/issues/580

I'm a little hesitant to also conflate control of OpenGL tools with "tiff-tools-unsupported" and feel like it could be its own variant. So right now people using `libtiff@4.5` may be bitten by #44545 but don't know if the edge case is worth the complication of two interacting variants.